tests/ops/test_qubit_ops.py

```diff
@@ -716,6 +728,34 @@ def test_isingxx_decomposition(self, tol):
 
         assert np.allclose(decomposed_matrix, op.matrix, atol=tol, rtol=0)
 
+    def test_isingyy_decomposition(self, tol):
+        """Tests that the decomposition of the IsingYY gate is correct"""
+        param = 0.1234
+        op = qml.IsingYY(param, wires=[3, 2])
+        res = op.decomposition(param, op.wires)
+
+        assert len(res) == 3
+
+        assert res[0].wires == Wires([3, 2])
+        assert res[1].wires == Wires([3])
+        assert res[2].wires == Wires([3, 2])
+
+        assert res[0].name == "CY"
+        assert res[1].name == "RY"
+        assert res[2].name == "CY"
+
+        mats = []
+        for i in reversed(res):
+            if i.wires == Wires([3]):
+                # RY gate
+                mats.append(np.kron(np.eye(2), i.matrix))
+            else:
+                mats.append(i.matrix)
+
+        decomposed_matrix = np.linalg.multi_dot(mats)
+
+        assert np.allclose(decomposed_matrix, op.matrix, atol=tol, rtol=0)
+
```

Comment:
I don't see any other tests of the operation?

Most importantly, we should be testing that gradients and adjoints are computed correctly!